doc/man/man3/Makefile.am

```diff
@@ -0,0 +1 @@
+dist_man_MANS = suite_create.3
```

Comment:
Is there a way for the man stuff to work with CMake as well?

Reply:
This is actually what I'm working on. I had previously only ever done this with Autotools, but I found [GNUInstallDirs](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html) while reading the documentation. Using this with the `INSTALL` target and the `CMAKE_INSTALL_PREFIX` option seems to be what we need.

I'll have to do more research to see how far back the functionality goes to see what the minim required CMake version is.

Reply:
Increasing the minimum CMake version is OK, if it is needed to pickup this feature.

Do you want to include the CMake support in this PR, or add it in a future PR?

Reply:
I think it would be better to include CMake support in the same pull request, for the sake of keeping installation functionality uniform regardless of the buildsystem. If anything, I'll get some good practice with CMake, and it'll help to get me even more familiar with the codebase.

Reply:
OK. Let me know when you want me to take a look at this PR again.

Reply:
Will do!

Reply:
@brarcher Hey, Branden, I finished implementing this functionality. The man pages can now be installed when using either Autotools or CMake. They're actually also defined as a separate (and optional) component in CMake, so they can be separately installed using `cmake --install . --component docs`, although only the sources contain this documentation at the moment, I need to edit the `README.md` to reflect these changes.

Reply:
Are the man pages normally installed with everything else? Or, must they be explicitly installed? That is, does there need to be an update to the GitHub actions to trigger this new path?

Reply:
They are currently included in the regular installation when you run `cmake --install .`.